client/jetpack-connect/remote-credentials.js

```diff
@@ -243,6 +251,11 @@ export class OrgCredentialsForm extends Component {
 						) }
 					</div>
 				</div>
+				<div className="jetpack-connect__note">
+					{ translate(
+						'Note: Your WordPress credentials are different than your WordPress.com credentials.'
```

Comment:
Not sure if this is a good suggestion or not, @jeffgolenski ?

```suggestion
						'Note: Your WordPress credentials can be different than your WordPress.com credentials.'
```

Reply:
Hmmm not sure. It's true they could be the same, but this is a tough one. Your call. ;-P

Reply:
Ahhhhh, I read that as “your username and password for wp-admin _can be_ different from your username and password you use on WordPress.com”. Just goes to show how nuanced these language differences are.

@michelleweber Do you have any suggestions here?

Reply:
@jeffgolenski Reverted this bit of copy because yours made a lot more sense and I had misinterpreted it initially.

Reply:
Which is correct, and what do we want to say? 

If the point of the note is to reassure people that they're using the right credentials, then I'd say "Your WP credentials may not be the same as your WP.com credentials."

If the point of the note is to say that WP and WP.com are not the same thing, then I'd say something more like "WordPress credentials are not the same as WP.com credentials. Be sure to enter the username and password for your self-hosted WordPress site."

It depends on what you're trying to communicate!

Reply:
I believe it's the latter, and I'll make that adjustment. Thanks @michelleweber !